datasets/Cargo.toml

```diff
@@ -14,6 +14,9 @@ ndarray-csv = "=0.5.1"
 csv = "1.1"
 flate2 = "1.0"
 ndarray-rand = { version = "0.14", optional = true }
+statrs = {version = "0.16.0", optional = true }
```

Comment:
I don't see this being used in the code

Reply:
The distributions are coming from that package. Specifically from this `use statrs::distribution::{Binomial, DiscreteUniform};`

Reply:
If the user wants those distributions then they can add the dependency in their own code. No need to include it in our crate.

Reply:
If it's used in the examples, then that's fine